packages/frontend-shared/src/components/Button.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <button
-    v-if="!props.href && !props.to"
+    v-if="((!props.href && !props.to) || (props.disabled && props.to))"
```

Comment:
Why add this extra conditional to render a button when the element should be disabled?  Was the disabled styling for the `a` tag not looking correct?  It would be better to always render an `a` tag when it is disabled.

Reply:
I still do not like adding this conditional here.  The more I look at the Button component, I think it is doing too much, but do not want to refactor too much for this story.  I did debug this and it is not because of the null `href`, but because the `RouterLink` is expecting a `to` prop which is not passed to it when disabled.  You can look at the `linkProps` computed around line 150.  

There are two solutions to try:
* Remove this conditional and go to the `linkProps` and ensure the `to` parameter is passed to the `RouterLink` even when disabled.  In this case, an additional class of `pointer-events-none` needs to get added along with `cursor-default` around line 139 when disabled.  This will prevent the link from being clickable.
* Look at the `linkVersion` variable and have it render a `BaseLink` when disabled instead of a `RouterLink`.  This will keep it rendering an `a` tag instead of switching to a `button`, but should render it without an href which means clicking it would do nothing.  

I am struggling with which option is the right solution for accessibility purposes.  @marktnoonan Do you have any suggestions here?  

Also, please add a test in Button.cy.tsx to test a disabled `Button` with a `to` prop.